docs/backup/05-01-velero.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Velero chart
+type: Configuration
+---
+
+To configure the Velero chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.volumeSnapshotLocation** | The location to store volume snapshots created for the backup. It is cloud provider-specific. For example, to set up `aws` as the provider of the snapshot storage location, override the value for the **global.volumeSnapshotLocation.spec.provider** parameter. [See](https://velero.io/docs/v1.0.0/api-types/volumesnapshotlocation/) the official Velero documentation for the full list of configurable parameters and examples. | None |
+| **global.backupStorageLocation** |  The location to store backups. It is cloud provider-specific. For example, to set up `aws` as the provider of the backup storage location, override the value for the **global.backupStorageLocation.spec.provider** parameter. [See](https://velero.io/docs/v1.0.0/api-types/backupstoragelocation/) the official Velero documentation for the full list of configurable parameters and examples. | None |
```

Comment:
Same simplification here: In general here is the docu: https://github.com/heptio/velero/blob/release-0.11/docs/api-types/backupstoragelocation.md

global.backupStorageLocation.name = cloud provider name
global.backupStorageLocation.bucket = name of the bucket to use in the object store for the backups
global.backupStorageLocation.prefix = The directory inside a storage bucket where backups are to be located

That are the supported attributes like described in docu
global.backupStorageLocation.config.region
global.backupStorageLocation.config. s3ForcePathStyle
global.backupStorageLocation.config. s3Url
global.backupStorageLocation.config. kmsKeyId
global.backupStorageLocation.config. resourceGroup
global.backupStorageLocation.config. storageAccount
global.backupStorageLocation.config. publicUrl